modules/efk-logging-deploy-subscription.adoc

```diff
@@ -27,9 +27,152 @@ The initial set of {product-title} nodes might not be large enough to support th
+----
+<1> You must specify the `openshift-operators-redhat` namespace.
+
+.. Create the Namespace object:
++
+----
+$oc create -f eo-project.yaml
+----
+
+.. Create a Subscription object YAML file (for example, `eo-sub.yaml`) to
+subscribe a namespace to an Operator. 
++
+.Example Subscription
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  generateName: elasticsearch-
+  namespace: openshift-operators
+spec:
+  source: elasticsearch
+  sourceNamespace: openshift-operators-redhat <1>
+  name: elasticsearch-operator
+channel: preview
+----
+<1> You must specify the `openshift-operators-redhat` namespace.
+
+. Create the Subscription object:
++
+----
+$ oc create -f eo-sub.yaml
+----
+
+.. Create a CatalogSourceConfig object YAML file (for example, `eo.csg.yaml`) to enable the Elasticsearch Operator on the cluster. 
++
+.Example CatalogSourceConfig
+[source,yaml]
+----
+apiVersion: "operators.coreos.com/v1"
+kind: "CatalogSourceConfig"
+metadata:
+  name: "elasticsearch"
+  namespace: "openshift-marketplace"
+spec:
+  targetNamespace: "openshift-operators-redhat" <1>
+  packages: "elasticsearch-operator"
+----
+<1> You must specify the `openshift-operators-redhat` namespace.
++
+The Operator generates a CatalogSource from your CatalogSourceConfig in the
+namespace specified in `targetNamespace`.
+
+.. Create the CatalogSourceConfig object:
++
+----
+$ oc create -f eo.csg.yaml
+----
+
+.. Create an Operator Group object YAML file (for example, `eo-og.yaml`) for the Elasticsearch operator:
++
+----
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: openshift-operators-redhat 
+  namespace: openshift-operators-redhat <1>
+spec: {}
+----
+<1> You must specify the `openshift-operators-redhat` namespace.
+
+.. Create the Operator Group object:
++
+----
+$ oc create -f eo.og.yaml
+----
+
+.. Change to the `openshift-operators-redhat` project:
++
+----
+oc project openshift-operators-redhat
+
+Now using project "openshift-operators-redhat"
+----
+
+.. Create a Role-based Access Control (RBAC) object file (for example, `eo-rbac.yaml`) to grant Prometheus permission to access the Elasticsearch namespace:
++
+----
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: prometheus-k8s
+  namespace: openshift-operators-redhat
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - services
+  - endpoints
+  - pods
+  verbs:
+  - get
+  - list
+  - watch
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: prometheus-k8s
+  namespace: openshift-operators-redhat
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: prometheus-k8s
+subjects:
+- kind: ServiceAccount
+  name: prometheus-k8s
+namespace: openshift-operators-redhat
+----
+
+.. Create the RBAC object:
++
+----
+$ oc create -f eo.rbac.yaml
+----
+
+. You can install the Cluster Logging Operator using the {product-title} web console for best results: 
```

Comment:
Do we need instructions for installing and deploying Cluster Logging using the CLI like above for Elasticsearch?

Reply:
@richm I left the console steps for cluster logging as it seems much cleaner and easier, especially where the user doesn't need any customizations. It does seem a bit incongruous to use different methods. What do you think?

Reply:
Do we document somewhere else in the openshift documentation about how to install/deploy components using the CLI?

Reply:
We do:
https://docs.openshift.com/container-platform/4.1/applications/operators/olm-adding-operators-to-cluster.html#olm-installing-operator-from-operatorhub-using-cli-olm-adding-operators-to-a-cluster

Reply:
Then we do not have to document it here.